pkg/virt-operator/kubevirt_test.go

```diff
@@ -512,8 +524,14 @@ var _ = Describe("KubeVirt Operator", func() {
 			injectMetadata(&obj.(*batchv1.Job).ObjectMeta, config)
 			addInstallStrategyJob(resource)
 		case *k8sv1.ConfigMap:
+			cm := obj.(*k8sv1.ConfigMap)
```

Comment:
use `&cm.ObjectMeta` instead of `&obj.(*k8sv1.ConfigMap).ObjectMeta` in the line below as you already made the assertion here